gdal/ogr/ogrsf_frmts/cad/GNUmakefile

```diff
@@ -0,0 +1,20 @@
+
+include ../../../GDALmake.opt
+
+SUBDIRS-yes := libopencad
+
+OBJ	= \
+	ogrcaddriver.o \
+	gdalcaddataset.o \
+	ogrcadlayer.o \
+	vsilfileio.o
+
+CPPFLAGS	:=	-I. -I.. -I../.. $(CPPFLAGS) -std=c++11 
```

Comment:
configure.in seems to have only part of the changes from configure. The source is configure.in. configure should be the result of running autoconf (2.69)

libopencad/GNUmakefile and libopencad/dwg/GNUmakefile should use $(CXX) and not $(CC)

I got a few compiler warnings :
r2000.cpp: In member function 'virtual int DWGFileR2000::createFileMap()':
r2000.cpp:761:66: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
cadclasses.cpp: In member function 'CADClass CADClasses::getClassByNum(short int) const':
cadclasses.cpp:70:13: warning: missing initializer for member '_class::sCppClassName' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::sApplicationName' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::sDXFRecordName' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::dProxyCapFlag' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::dInstanceCount' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::bWasZombie' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::bIsEntity' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::dClassNum' [-Wmissing-field-initializers]
cadclasses.cpp:70:13: warning: missing initializer for member '_class::dClassVersion' [-Wmissing-field-initializers]

Regarding MSVC support, I see that cad is listed unconditionnaly in ogr/ogrsf_frmts/makefile.vc. Do you know which is the minimum MSVC version (or at least a known good version) that supports it ? It should probably be conditionnalized compiled to it.